README.md

```diff
@@ -289,7 +289,7 @@ print("nef is super cool: \(example)")
 // nef is super cool: This is an example
 ```
 
-Running the following command we will customize the `background color` to ![#d54048](https://placehold.it/15/d54048/000000?text=+) bow, `hide the number of lines` and set the export file to `size 3`:
+Running the following command we will customize the `background color` to ![#d54048](https://placehold.it/15/d54048/000000?text=+) bow, `hides the number of lines` and set the export file to `size 3`:
```

Comment:
It does not have the `s` since it is preceded by `will`.
```suggestion
Running the following command we will customize the `background color` to ![#d54048](https://placehold.it/15/d54048/000000?text=+) bow, `hide the number of lines` and set the export file to `size 3`:
```